packages/apps/src/Endpoints/Network.tsx

```diff
@@ -58,10 +58,10 @@ function NetworkDisplay ({ apiUrl, className = '', setApiUrl, value: { isChild,
               {isRelay
                 ? t('Relay chain')
                 : paraId && paraId < 1000
-                  ? t('{{relay}} System', { replace: { relay } })
+                  ? t('{{relay}} ', { replace: { relay } })
                   : paraId && paraId < 2000
-                    ? t('{{relay}} Common', { replace: { relay } })
-                    : t('{{relay}} Parachain', { replace: { relay } })
+                    ? t('{{relay}} ', { replace: { relay } })
+                    : t('{{relay}} ', { replace: { relay } })
```

Comment:
nitpick: not sure what this for...

Reply:
On the explorer, when you click on a chain, say `sparknet` it used to show `CORD Network parachain` as a description. now with latest commit, it shows `CORD Network testnet`

Reply:
Ah! Ok. Then this is a must have fix.

Reply:
Yes and this is done